src/components/ConnectionStatus.svelte

```diff
@@ -1,5 +1,8 @@
 <script lang="ts">
-  export let connected = false;
```

Comment:
The connection is out of context for this story. Need to create new story to define saving the state for the network and key so that we can access them throughout the app.

Reply:
This related component has it's own story/issue (https://github.com/LibertyDSNP/provider-dashboard/issues/106).  You can keep it in this PR but we'll need to update in a different PR.